lib/db/pokemon_grabber.py

```diff
@@ -0,0 +1,70 @@
+import requests
+
+
+# A method to get all gen 1 pokemon from pokeapi.co.
+# DOCUMENTATION:
+# RETURNS:
+# boolean error OR json species
+def get_pokemon_species():
+    API = "https://pokeapi.co/api/v2/generation/1/"
+    print(f"[API] Getting pokemon species from: {API}")
+    res = requests.get(API)
+
+    if res.status_code != 200:
+        print(f"[API] Failed getting pokemon species from: {API}")
+        return False
+    else:
+        return res.json()["pokemon_species"]
+
+
+# A method to get the selected pokemon's attributes.
+# DOCUMENTATION:
+# int pokemon_API_ID
+# RETURNS:
+# boolean error OR dict attributes
+def get_a_pokemons_attributes(pokemon_API_ID):
+    API = "https://pokeapi.co/api/v2/pokemon-species/" + str(pokemon_API_ID) + "/"
+    print(f"[API] Getting pokemonID {pokemon_API_ID}'s attributes from: {API}")
+    res = requests.get(API)
+
+    if res.status_code != 200:
+        print(f"[API] Failed getting pokemon's attributes from: {API}")
+        return False
+    else:
+        res_json = res.json()
+        return {
+            "pokedex_ID":   res_json["pokedex_numbers"][0]["entry_number"],
+            "name":         res_json["name"],
+            "evolves_from": res_json["evolves_from_species"]
```

Comment:
Need to consider how single stage evolutions and first stages are handled. They will have no previous (e.g. Charmander, Abra etc)

Reply:
We can discuss all of that in the discussion thread. Currently, single evolution and first stages are represented as NULL in this field within the db.
If it does have a 'evolves from' species, perhaps storing the pokeID/pokedex number or even the api link to said pokemon could be a better bit of data.